.github/workflows/run_spec_tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Ruby Test Env
+# As opposed to the ci env used before on the Ruby action
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Let's definitely add the 'on PR' option

Reply:
Done.

Reply:
Just seen this actually happen on updating this PR: https://github.com/wearefuturegov/beacon/actions/runs/69970929